### PR TITLE
Added @idearium/log/multi

### PR DESCRIPTION
**Pull request by smebberson**
This PR adds `@idearium/log/multi` to make logging data dynamic based on `LOG_LEVEL`.

## Related PRs

- idearium/ras-rsso#177
- idearium/rest-image#32
- idearium/idearium-lib#348
- idearium/idearium-lib#349

## Test

- [x] Review the code.
- [x] Review the logs.
- [x] Ensure the tests and passing and complete enough.
- [ ] Complete `@idearium/log/multi` testing through the steps outlined in idearium/ras-rsso#177.

## Deployment

- [ ] Version bump.

### CODE REVIEW

File: packages/log/multi.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+        log[loggedLevel](composedData, message);
+
+        return { data: composedData, level: loggedLevel };
+    };
+
+    return multiLogger;
+};
+
+module.exports = (options = {}) => {
```
Comment (allanchau) [inline packages/log/multi.js:38]: Can you please turn this into
```js
const log = (options = {}) => {
...
};

module.exports = log;
```

**Review by allanchau — CHANGES_REQUESTED** [review]
@smebberson Small change needed here, otherwise looks good to me

File: packages/log/multi.js (hunk 1 above)
Comment (smebberson) [reply packages/log/multi.js]: @allanchau, I renamed this.

**Comment by smebberson** [comment]
@allanchau, are you okay for me to release now?

**Review by allanchau — APPROVED** [review]

**Comment by allanchau** [comment]
@smebberson Yep all good now

**Pull request merged**